### PR TITLE
Enable Omniauth Google and Facebook

### PR DESCRIPTION
**Pull request by LauraJaimeC**

### CODE REVIEW

File: config/application.example.yml [hunk 1]
```diff
@@ -15,6 +15,11 @@ default: &default
   # GeoCoder configuration (developer.here.com)
   HERE_API_KEY: "your here api key"
   SECRET_KEY_BASE: "secret_key_base"
+  # Omniauth credentials
+  OMNIAUTH_FACEBOOK_APP_ID: 'fb_app_id'
+  OMNIAUTH_FACEBOOK_APP_SECRET: 'fb_app_secret'
+  OMNIAUTH_GOOGLE_CLIENT_ID: 'google_client_id'
+  OMNIAUTH_GOOGLE_CLIENT_SECRET: 'google_client_secret'
```
Comment (tramuntanal) [inline config/application.example.yml:22]: can we add this in `config/initializers/figaro.rb`?

Comment (LauraJaimeC) [reply config/application.example.yml:22]: `figaro.rb` doesn't exist :thinking:

Comment (LauraJaimeC) [reply config/application.example.yml:22]: Done

File: config/initializers/figaro.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+env= Rails.env
+keys = %w{ SECRET_KEY_BASE }
+unless env.development? or env.test?
+  keys += %w{ DB_DATABASE DB_PASSWORD DB_USERNAME }
```
Comment (tramuntanal) [inline config/initializers/figaro.rb:4]: this keys are always required, this line should be outside the `unless`

**Review by tramuntanal — CHANGES_REQUESTED** [review]

**Review by tramuntanal — APPROVED** [review]
Great @LauraJaimeC !

**Pull request merged**